README.md

```diff
@@ -83,26 +94,37 @@ compiled_model = core.compile_model(ov_model, 'CPU')
 data = np.random.rand(1, 224, 224, 3)
 output = compiled_model({0: data})
 ```
+</details>
+
+
+## Documentation
+
+[User documentation](https://docs.openvino.ai/) contains detailed information about OpenVINO and guides you from installation through optimizing and deploying models for your AI applications.
 
-OpenVINO also supports CPU, GPU, and NPU devices and works with models in TensorFlow, PyTorch, ONNX, TensorFlow Lite, PaddlePaddle model formats.
-With OpenVINO you can do automatic performance enhancements at runtime customized to your hardware (preserving model accuracy), including:
-asynchronous execution, batch processing, tensor fusion, load balancing, dynamic inference parallelism, automatic BF16 conversion, and more.
```

Comment:
Why do we remove to mention all FWs? Strongly disagree.

Reply:
to remove the fluff after it :) added links to devices,formats and APIs instead
pls check